path_static_roles.go

```diff
@@ -43,7 +64,7 @@ func (b *backend) pathListStaticRoles() []*framework.Path {
 func (b *backend) pathStaticRoles() []*framework.Path {
 	return []*framework.Path{
 		{
-			Pattern: staticRolePath + framework.GenericNameRegex("name"),
+			Pattern: strings.TrimSuffix(staticRolePath, "/") + GenericNameWithForwardSlashRegex("name"),
```

Comment:
nit, it is a bit odd that we trimming a constant but I guess it is easier than decomposing `staticRolePath` ?